frontend/src/metabase/home/components/EmbedHomepage/EmbedHomepage.tsx

```diff
@@ -27,9 +32,9 @@ export const EmbedHomepage = () => {
 
   return (
     <EmbedHomepageView
-      exampleDashboardId={1}
-      embeddingAutoEnabled={false}
-      licenseActiveAtSetup={true}
+      exampleDashboardId={exampleDashboardId}
+      embeddingAutoEnabled={embeddingAutoEnabled}
+      licenseActiveAtSetup={licenseActiveAtSetup}
       plan="oss-starter"
```

Comment:
the `plan` prop will get changed in the [next pr](https://github.com/metabase/metabase/pull/40528)